src/khmer/_cpy_hashtable.cc

```diff
@@ -464,52 +594,42 @@ hashtable_consume_seqfile_banding_with_mask(khmer_KHashtable_Object * me, PyObje
 }
 
 PyObject *
-hashtable_consume_seqfile_with_reads_parser(khmer_KHashtable_Object * me,
-        PyObject * args)
+hashtable_consume_seqfile_banding_with_mask_with_reads_parser(khmer_KHashtable_Object * me, PyObject * args)
```

Comment:
Looks like git is struggling with the similarity between all of the variants of this function and the fact that I cut n pasted this function further up, again for consistency.